documentation/Installation.md

```diff
@@ -3837,6 +3837,7 @@ For example:
     config_map:
       server-tokens: "False"
 ```
+**Warning**: Ingress-nginx and HAproxy use proxy protocol in the default configuration. If you are using a load balancer without a proxy protocol, it should also be disabled in ingress-nginx. To do this, specify `use-proxy-protocol: "false"` in configmap.
```

Comment:
We need to add a default `config_map` settings, since it's important change in configuration.

Reply:
Added